dtests/build.gradle

```diff
@@ -89,16 +96,17 @@ archivesBaseName = 'gemfirexd-scala-tests'
 
 testClasses.doLast {
     copyTestsCommonResources(buildDir)
-}
-
-copy {
-    def osDir = osName.getFamilyName().replace(" ", "").toLowerCase()
-    from (rootDir.getAbsolutePath() + '/dtests/src/test/java') {
-        include '**/*.bt'
-        include '**/*.conf'
-        include '**/*.inc'
-        include '**/*.sql'
+    copy {
+        def osDir = osName.getFamilyName().replace(" ", "").toLowerCase()
```

Comment:
Sumedh made this change already as part of some other PR. Why are you merging it like this? This corrupts the history of git. You should rather merge master on your branch and you will get this change.